bodhi/server/models.py

```diff
@@ -786,6 +786,12 @@ class ComposeState(DeclEnum):
     Column('user_id', Integer, ForeignKey('users.id')),
     Column('package_id', Integer, ForeignKey('packages.id')))
 
+package_release_table = Table(
+    'package_release_table', Base.metadata,
+    Column('package_id', Integer, ForeignKey('packages.id'), nullable=False),
+    Column('release_id', Integer, ForeignKey('releases.id'), nullable=False)
+)
```

Comment:
Let's give this table a more descriptive name, because it doesn't sound like it is related to critical path packages. How about ```critical_path_packages_table```?